### ISSUE TITLE
As a SVF newcomer, I encountered problems while running the tutorial.

### ISSUE THREAD

**Issue by liushengahn** [SVF-tools/SVF#1201]
I ran the examples in the tutorial to obtain the call graph.
```
void swap(char **p, char **q){

  char* t = *p; 
       *p = *q; 
       *q = t;
}
int main(){
      char a1, b1; 
      char *a = &a1;
      char *b = &b1;
      swap(&a,&b);
}
```
I didn't get the result in the picture. Instead, there was an extra "malloc" call point. I didn't modify the program. I don't know why.
![image](https://github.com/SVF-tools/SVF/assets/46082729/0ef5618f-3ac0-409b-a531-c7513b12a2ad)
I used SVF to analyze an empty file, but 'malloc' still appeared.
![image](https://github.com/SVF-tools/SVF/assets/46082729/a1c8fc56-cd82-4947-bab1-6acb22e2fa45)
Did I make a mistake in that step?
Looking forward to your answers very much. Thank you!

**Comment by yuleisui** [comment]
@shuangxiangkan could you answer this question

**Comment by shuangxiangkan** [comment]
Please try the latest SVF. This issue has already been fixed in #1204

**Comment by liushengahn** [comment]
thank you.
